04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,69 @@
+const Product = require("../models/product");
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .sort("name")
+    .select("name price");
+
+  res.status(200).json({ products, nbHits: products.length });
+};
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+  if (featured) {
+    queryObject.featured = featured === "true" ? true : false;
```

Comment:
```suggestion
    queryObject.featured = !!featured;
```

Much simpler wya of writing the same thing